lib/plugins/aws/remove/lib/stack.js

```diff
@@ -26,8 +27,47 @@ module.exports = {
       .then(() => cfData);
   },
 
+  disassociateUsagePlan() {
```

Comment:
Sounds good. Thanks for looking into it. 👍 

I'll also take a deeper dive into it next week. Maybe we can uncover something there...